src/game/MainMenuScreen.cc

```diff
@@ -84,7 +84,7 @@ static void RestoreButtonBackGrounds(void);
 
 ScreenID MainMenuScreenHandle(void)
 {
-	if (guiSplashStartTime + 4000 > GetJA2Clock())
+	if (guiSplashStartTime + INTRO_SPLASH_DURATION > GetJA2Clock())
```

Comment:
Isn't this redundant now? `guiSplashStartTime` is set before `splashDisplayedMoment` (it is set in `InitJA2SplashScreen` almost immediately after `InitializeJA2Clock`, so it should always be 0 or maybe 1 on slower machines).